### DISCUSSION TITLE
Help with DGT

### DISCUSSION THREAD

**Discussion by Felipefnol** [Telecominfraproject/oopt-gnpy#460] [Q&A] [answered]
Help with DGT!

We are trying to insert an EDFA we have here in our research into GNPY. We got the DGT based on the article cited in the reference ("The Er3+ -Fiber Gain Coefficient Derived from a Dynamic Gain Tilt Technique.").

We would like to make sure that we have done the DGT calculation the right way.
In the GNPY documentation we have 2 examples of DGT (Default_EDFA_config and Juniper-BoosterHG).

Can I find the used information (Gain 1, Gain 2, Wavelength Reference) of one of these EDFA (Default_EDFA_config and Juniper-BoosterHG) in some GNPY documentation ?

This would be the most reliable way for me to reproduce the results and find the correct DGT value.

**Comment by Felipefnol** [comment]
![image](https://user-images.githubusercontent.com/30154886/217095552-7ea3f982-a4b3-4682-ab0f-4bda3aae0aef.png)

I believe I did the correct procedure. I would like to be sure so that I can publish our results. 

If someone can help me in this regard, I appreciate it.

**Reply by jktjkt** [reply]
@AndreaDAmico mind taking a look on the DGT derivation here?

**Reply by AndreaDAmico** [reply, marked as answer]
As we discuss directly with @Felipefnol, that is the right procedure to evaluate the DGT, as reported in R. D. Muro, “The er3+-fiber gain coefficient derived from a dynamic gain tilt technique”, JLT, vol. 18, no. 3, pp. 343–347, 2000. DOI: 10.
1109/50.827506. 
After an accurate investigation, we discover that the current implementation of the EDFA gain tilt profile in GNPy is not correct (see #471).
Furthermore, it can be shown that the DGT is not the most convenient quantity to consider in order to reproduce the tilt-dependent gain ripples, therefore we are including an advance implementation of the EDFA which accurately evaluate this quantity.

**Reply by Felipefnol** [reply]
Thank you for your clarifications.
